moveit_setup_assistant/templates/moveit_config_pkg_template/launch/demo.launch

```diff
@@ -55,7 +55,7 @@
 
   <!-- Run Rviz and load the default config to see the state of the move_group node -->
   <include file="$(dirname)/moveit_rviz.launch" if="$(arg use_rviz)">
-    <arg name="rviz_config" value="$(find [GENERATED_PACKAGE_NAME])/launch/moveit.rviz"/>
+    <arg name="rviz_config" value="$(dirname)/moveit.rviz"/>
```

Comment:
you are right, it actually works due to our file layout - even though the use is buggy and does not set a good example I guess.. .  https://github.com/ros-planning/moveit/pull/2748#issue-681145812

Reply:
You are right. I ran into this bug recently as well. I will revert this change.

Reply:
TMYK